WalletWasabi.Gui/Controls/WalletExplorer/SendControlViewModel.cs

```diff
@@ -61,6 +62,9 @@ public abstract class SendControlViewModel : WasabiDocumentTabViewModel, IWallet
 
 		private const string WaitingForHardwareWalletButtonTextString = "Waiting for Hardware Wallet...";
 
+		private static readonly FeeRate MinRelayTxFeeRate = new FeeRate(Money.Satoshis(1), 1);
```

Comment:
hmm, this is not the same as before. What was the reason of the change? I was trying to enter 0.5 sat/vbyte and it is invalid.

Reply:
I thought zero was an invalid value for sats/vbyte?